econml/dml.py

```diff
@@ -25,242 +25,7 @@
-
-    def const_marginal_effect(self, X=None):
-        """
-        Calculate the constant marginal CATE θ(·).
-
-        The marginal effect is conditional on a vector of
-        features on a set of m test samples {Xᵢ}.
-
-        Parameters
-        ----------
-        X: optional (m × dₓ) matrix
-            Features for each sample.
-            If X is None, it will be treated as a column of ones with a single row
-
-        Returns
-        -------
-        theta: (m × d_y × dₜ) matrix
-            Constant marginal CATE of each treatment on each outcome for each sample.
-            Note that when Y or T is a vector rather than a 2-dimensional array,
-            the corresponding singleton dimensions in the output will be collapsed
-            (e.g. if both are vectors, then the output of this method will also be a vector)
-        """
-        if X is None:
-            assert self._d_x == (1,), "X was not None when fitting, so can't be none for effect"
-            X = np.ones((1, 1))
-        return self._model_final.predict(X)
-
-    def const_marginal_effect_interval(self, X=None, *, alpha=0.1):
-        if X is None:
-            assert self._d_x == (1,), "X was not None when fitting, so can't be none for effect"
-            X = np.ones((1, 1))
-        return super().const_marginal_effect_interval(X, alpha=alpha)
-
-    def effect_interval(self, X=None, T0=0, T1=1, *, alpha=0.1):
-        if X is None:
-            n_rows = shape(T0)[0] if ndim(T0) > 0 else 1
-            assert self._d_x == (1,), "X was not None when fitting, so can't be none for effect"
-            X = np.ones((n_rows, 1))
-        return super().effect_interval(X, T0=T0, T1=T1, alpha=alpha)
-
-    def score(self, Y, T, X=None, W=None):
-        X, W = self._check_X_W(X, W, Y)
-        X, T = self._expand_treatments(X, T)
-        if T.ndim == 1:
-            T = reshape(T, (-1, 1))
-        if Y.ndim == 1:
-            Y = reshape(Y, (-1, 1))
-        Y_test_pred = np.zeros(shape(Y) + (self._n_splits,))
-        T_test_pred = np.zeros(shape(T) + (self._n_splits,))
-        for ind in range(self._n_splits):
-            if self._discrete_treatment:
-                T_test_pred[:, :, ind] = reshape(self._models_t[ind].predict(X, W)[:, 1:], shape(T))
-            else:
-                T_test_pred[:, :, ind] = reshape(self._models_t[ind].predict(X, W), shape(T))
-            Y_test_pred[:, :, ind] = reshape(self._models_y[ind].predict(X, W), shape(Y))
-        Y_test_pred = Y_test_pred.mean(axis=2)
-        T_test_pred = T_test_pred.mean(axis=2)
-        Y_test_res = Y - Y_test_pred
-        T_test_res = T - T_test_pred
-        effects = reshape(self._model_final.predict(X), (-1, shape(Y)[1], shape(T)[1]))
-        Y_test_res_pred = reshape(np.einsum('ijk,ik->ij', effects, T_test_res), shape(Y))
-        mse = ((Y_test_res - Y_test_res_pred)**2).mean()
-        return mse
-
+from ._rlearner import _RLearner
```

Comment:
Am I missing something, or is this _rlearner file not present in this set of commits?